src/components/OrgPicker/OrgPicker.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * While visually similar to the USWDS Combo box component, this expandable element presents a scrollable list of links. The link at the bottom of the menu directs the user to a page listing all the organizations they can access.
+ */
+import React from 'react';
+// import { OrgObj } from '@/api/cf/cloudfoundry-types';
+import { Button } from '@/components/uswds/Button';
+import { useState } from 'react';
+import Image from 'next/image';
+import expandIcon from '@/../public/img/uswds/usa-icons/expand_less.svg';
+import collapseIcon from '@/../public/img/uswds/usa-icons/expand_more.svg';
+import { OrgPickerList } from './OrgPickerList';
+import { OrgPickerFooter } from './OrgPickerFooter';
+
+export function OrgPicker({ single }: { single: Boolean }) {
+  const [toggle, setToggle] = useState(true);
+
+  return !single ? (
+    <div className="display-block desktop:display-flex">
+      <span className="usa-label font-sans-xs padding-right-105">
+        Current organization:
+      </span>
+      <nav className="orgs-selector width-mobile border border-base-light shadow-2 font-sans-xs padding-x-105 margin-y-105">
```

Comment:
@cannandev, this is really looking fantastic, and it works even better.

A few design notes, if they’re helpful!

1. I think a `bg-white` class might be missing here? The list is currently showing up without a background for me.
2. The shadow should only be applied when the list is expanded. Is that possible?
3. Would it be possible to tighten up the shadow a bit? It’s a little bigger / more diffuse than I was hoping it’d be. (Screenshot attached of what I was aiming for; but/and: let me know if I need to rethink the design!)
4. The menu feels just a _tiny_ bit bigger than I was hoping for. The two main things I noticed:
  - The font size is around 16px; I was aiming for 15px. (Or the relative equivalent!)
  - Possibly related to ^that point, the collapsed state of the menu feels a bit tall. (I also noticed the vertical padding’s a little bigger than in the design: 12px vs 8px on widescreen; 12px vs 6px on small)

Let me know if anything above is worth talking through a bit more! And thanks again, Claire. This really is looking fantastic — excited to see this land in the prototype!

<img width="499" alt="Screenshot 2024-09-05 at 1 12 08 PM" src="https://github.com/user-attachments/assets/4a292bd7-2a26-4cb8-85b6-93bb7633466a">
<img width="243" alt="Screenshot 2024-09-05 at 1 13 32 PM" src="https://github.com/user-attachments/assets/a54c201a-a3bc-4e5a-a194-428ea6a5ad6e">

Reply:
Awesome feedback. I'll work on tightening this up today!